workspaces/redhat-argocd/packages/app/package.json

```diff
@@ -36,6 +36,7 @@
     "@backstage/plugin-catalog-graph": "^0.4.22",
     "@backstage/plugin-catalog-import": "^0.13.4",
     "@backstage/plugin-catalog-react": "^1.20.1",
+    "@backstage/plugin-kubernetes": "^0.12.10",
```

Comment:
You're adding the `@backstage/plugin-kubernetes` frontend dependency back into the Argo CD plugin. Is this addition intentional?
 
Just to share some context,  as part of https://github.com/janus-idp/backstage-plugins/pull/1869, we specifically worked on removing the `@backstage/plugin-kubernetes` frontend dependency from the Tekton and Topology plugins.